topics/how-to.md

```diff
@@ -412,7 +412,7 @@ If you do not want to use the bundled backup functionality or need manual contro
     * `<TeamCity Data Directory>/system/artifacts` (optional) if you want build artifacts and build logs (including tests failure details) to be preserved on the new server
     * `<TeamCity Data Directory>/system/changes` (optional) if you want personal changes to be preserved on the new server
     * `<TeamCity Data Directory>/system/pluginData` (optional) if you want to preserve the state of various plugins, build triggers, and settings audit diff
-    * `<TeamCity Data Directory>/system/caches` (optional) is not necessary to copy to the new server. If not copied, it will be rebuilt on startup, but that can take some time.
+    * `<TeamCity Data Directory>/system/caches` and `<TeamCity Data Directory>/system/caches_main` (optional) are not necessary to copy to the new server. If not copied, they will be rebuilt on startup, but that can take some time (expect some slow down)
```

Comment:
Please change to
`<TeamCity Data Directory>/system/caches` and `<TeamCity Data Directory>/system/caches_main` (optional) do not need to be copied to the new server. If not copied, they will be rebuilt on startup and it may take some time.